scalableminds/webknossos
# Add snapshot test: public annotations viewable while logged out

This was broken for three days, fixed in #5193 – but we should have noticed sooner.

### Steps to test:
- CI run should be enough (I also tested that if I set the annotation visibility to Internal in the csv, the test does fail)

### Issues:
- fixes #5191 

------
